File: result/translate/transcoder/python2java/java/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.java
public static int f_gold ( int [ ] arr , int n ) {
  int res = 0 ;
  Map < Integer , Integer > m = new HashMap < Integer , Integer > ( ) ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    int Sum = 0 ;
    for ( int j = i ;
    j < n ;
    j ++ ) {
      Sum += arr [ j ] ;
      m . put ( Sum , m . get ( Sum ) + 1 ) ;
    }
  }
  for ( Integer x : m . keySet ( ) ) {
    if ( m . get ( x ) == 1 ) res += x ;
  }
  return res ;
}

File: result/translate/starchat/java2python/python/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.py
def f_gold(arr, n):
        res = 0
        m = {}
        for i in range(n):
            sum = 0
            for j in range(i, n):
                sum += arr[j]
                if sum in m:
                    m[sum] += 1
                else:
                    m[sum] = 1
        for x in m.items():
            if x[1] == 1:
                res += x[0]
        return res